web/src/app/workplace/components/task-list-toolbar/task-list-toolbar.component.ts

```diff
@@ -77,6 +78,13 @@ export class TaskListToolbarComponent implements OnInit {
     }
   }
 
+  filterWorkbasketNames() {
```

Comment:
The filter function doesn't work if you type in numbers.. It seems to work if you're using the following function. 

  filterWorkbasketNames() {
    this.filteredWorkbasketNames = this.workbasketNames.filter((value) =>
    value.toLowerCase().includes(this.resultName.toLowerCase())
    );
  }

Reply:
It worked with numbers but the user had to fill in the name from its beginning, e.g. 'basxet0' wasn't found when 'xet0' was typed in. Your solution solves this problem, thank you!

Reply:
:) 
The filterValue variable seems to be unused now..

Reply:
yes 😄